scripts/updateChromium.js

```diff
@@ -1,9 +1,11 @@
 const https = require('https');
 const { writeFileSync, readFileSync } = require('fs');
 const { execSync } = require('child_process');
-const BrowserFetcher = require('./lib/browserFetcher');
+const BrowserFetcher = require(`${__dirname}/../lib/browserFetcher`);
 const supportedPlatforms = BrowserFetcher.supportedPlatforms;
 
+const PACKAGE_JSON = `${__dirname}/../package.json`;
```

Comment:
These changes just make the script idempotent from where and how it is invoked.

Reply:
Can we use path.join for this?

```suggestion
const PACKAGE_JSON = path.join(__dirname, "..", "package.json");
```